fortify-issue-manager/src/main/java/com/microfocus/security/automation/fortify/issue/manager/ConfigurationManager.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.microfocus.security.automation.fortify.issue.manager;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ConfigurationManager {
```

Comment:
This class just has helper functions which can be static. The class can be final.

Reply:
I've now made this final and static, additional changes to support testing have also been added.